buildscripts/ci/tools/make_build_number.sh

```
@@ -0,0 +1,37 @@
+# along with this program.  If not, see <https://www.gnu.org/licenses/>.
+ARTIFACTS_DIR="build.artifacts"
+
+NUMBER=$1
+if [ -z "$NUMBER" ]; then
+    NUMBER="$(date -u +%y%m%d%H%M)"
+    NUMBER="${NUMBER:0:9}" # less than 2147483647 to fit in Int32 for WiX packaging tool
+fi
+
+if [ -z "$NUMBER" ]; then echo "error: not set BUILD_NUMBER"; exit 1; fi
+
+if [ -z "$2" ]; then OUT_DIR=$ARTIFACTS_DIR/env; fi
```

Shellcheck (0.11.0)

[warning] OUT_DIR appears unused. Verify use (or export if used externally).

(SC2034)